### PR TITLE
Graph component

### PR DESCRIPTION
**Pull request by soujay**
This PR includes the following changes
- Basic graph component with and without datapoints
- typo fixes

### CODE REVIEW

File: src/ServicePulse.Host/vue/src/assets/main.css
Comment (cquirosj) [inline src/ServicePulse.Host/vue/src/assets/main.css:1]: Are these changes required for the goal of your PR? I think they are going to revert some of the changes from Travis if you include this file.

File: src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue
Comment (cquirosj) [inline src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue:1]: Same: Are these changes required for the goal of your PR? I think they are going to revert some of the changes from Travis if you include this file.

**Review by cquirosj — CHANGES_REQUESTED** [review]

File: src/ServicePulse.Host/vue/src/assets/main.css [hunk 1]
```diff
@@ -275,7 +275,7 @@ pre {
 }
 
 .loading {
-    background-image: url('../img/loader_spinner.gif');
+    background-image: url('../a/img/loader_spinner.gif');
```
Comment (TravisNickels) [inline src/ServicePulse.Host/vue/src/assets/main.css:278]: Since this is the main.css for Vue it feels awkward to reference an image from the angular website.  We are eventually going to be removing all of the angular stuff, so I think it would be better for us to copy the image that we need to the Vue assets folder.  This exact line has already been updated in the feature branch to '../assets/loader_spinner.gif'

File: src/ServicePulse.Host/vue/src/assets/main.css [hunk 2]
```diff
@@ -2395,7 +2395,7 @@ footer span.connection-failed.ng-scope {
 
 
 .pa-connection-establishing {
-    background-image: url('../img/loader_spinner.gif');
+    background-image: url('../a/img/loader_spinner.gif');
```
Comment (TravisNickels) [inline src/ServicePulse.Host/vue/src/assets/main.css:2398]: We should copy this image to the Vue assets folder

File: src/ServicePulse.Host/vue/src/assets/main.css [hunk 3]
```diff
@@ -2674,7 +2674,7 @@ span.fa-exclamation-triangle.danger {
     width: 100%;
     height: 90vh;
     margin-top: -100px;
-    background-image: url('../img/sp-loader.gif');
+    background-image: url('../a/img/sp-loader.gif');
```
Comment (TravisNickels) [inline src/ServicePulse.Host/vue/src/assets/main.css:2677]: We should copy this image to the Vue assets folder

File: src/ServicePulse.Host/vue/src/assets/main.css (hunk 1 above)
Comment (soujay) [reply src/ServicePulse.Host/vue/src/assets/main.css]: There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .

File: src/ServicePulse.Host/vue/src/assets/main.css (hunk 2 above)
Comment (soujay) [reply src/ServicePulse.Host/vue/src/assets/main.css]: There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .

File: src/ServicePulse.Host/vue/src/assets/main.css (hunk 3 above)
Comment (soujay) [reply src/ServicePulse.Host/vue/src/assets/main.css]: There are a many such image files in the vue main css. I was planning to have all of them copied over late on. For now, I have copied the images of the ones mentioned in this PR .

File: src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue
Comment (soujay) [reply src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue:1]: No, changing the css was not the main goal of this PR. The changes to the image url was because of a js error that I was receiving, Since the other img paths in this same css document was from the angular folder, I maintained the same here are as well.

File: src/ServicePulse.Host/vue/src/assets/main.css
Comment (soujay) [reply src/ServicePulse.Host/vue/src/assets/main.css:1]: see belwo

**Comment by soujay** [comment]
I am abandoning this PR and created a new one - https://github.com/Particular/ServicePulse/pull/1619
